wob.c

```diff
@@ -598,13 +604,17 @@ main(int argc, char **argv)
 
 	wob_pledge();
 
-	argb_color background_color = BLACK;
-	argb_color bar_color = WHITE;
-	argb_color border_color = WHITE;
+	struct wob_colors colors = {
+		.background = BLACK,
+		.bar = WHITE,
+		.border = WHITE,
+	};
+
+	struct wob_colors old_colors = {0, 0, 0};
```

Comment:
This is equivalent to above:

```suggestion
	struct wob_colors old_colors = {0};
```

Reply:
fixed.